### PR TITLE
[FIX+IMP] purchase_discount: Closes #115 + improve documentation

### LINKED ISSUE #115: Discount doesn't goes to the invoice if odoo v8 is configured as invoicing based to incoming shipments.
When Settings => Purchases => Default invoicing control method => is set to "Based to incoming shipments", the discount is set to "0" in the invoice.

In other configurations ("order purchases" or "order lines") it runs ok.

Thank you.

### PR DESCRIPTION
**Pull request by pedrobaeza**
Closes #115

### CODE REVIEW

**Comment by yvaucher** [comment]
:+1:

File: purchase_discount/README.rst [hunk 1]
```diff
@@ -0,0 +1,39 @@
+.. image:: https://img.shields.io/badge/licence-AGPL--3-blue.svg
+    :alt: License
```
Comment (yvaucher) [inline purchase_discount/README.rst]: To replace with last changes on templates
https://github.com/OCA/maintainer-tools/commit/ae4f0605c72bbe0ca66a383f4584f9e750ef7dec

Comment (pedrobaeza) [inline purchase_discount/README.rst]: Done

**Comment by jgrandguillaume** [comment]
:+1:

**Pull request merged**